### PR TITLE
chore(suite-common): device middleware

### PR DESCRIPTION
**Pull request by juriczech**
## Related Issue

Resolve

### CODE REVIEW

**Review by PeKne — APPROVED** [review]

**Review by PeKne — APPROVED** [review]

File: suite-native/device/src/deviceMiddleware.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+export const prepareDeviceMiddleware = createMiddlewareWithExtraDeps(
+    (action, { dispatch, next }) => {
+        if (action.type === DEVICE.DISCONNECT) {
+            dispatch(forgetDisconnectedDevices(action.payload));
+        }
+
+        // propagate action to reducers
+        next(action);
```
Comment (Nodonisko) [inline suite-native/device/src/deviceMiddleware.ts:42]: Why is this here? Always put it on the end unless some very specific use cases that must be explicitely desribed in comment then.
